src/main/java/core/basesyntax/service/files/FileReaderCsvImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.files;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderCsvImpl implements FileReaderCsv {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("can't open the file", e);
```

Comment:
The error message in the catch block should be more informative. For example, it could include the file path that couldn't be opened. It would make debugging easier if such a situation occurs.